app/api/chat/route.ts

```diff
@@ -128,6 +129,7 @@ export async function POST(request: NextRequest) {
       onError: (e) => {
         sendErrorNotification({
           ...body,
+          path: '/api/chat',
```

Comment:
do you ever pass a variable other than `/api/chat`?
- if no, what information would be lost by removing the variable and hard coding it in your notification text?